Example/Example/Source/DynamicSize/DynamicImageSizeCell.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  DynamicImageSizeCell.swift
+//  Example
+//
+//  Created by Le VanNghia on 7/10/15.
+//  Copyright (c) 2015 Le Van Nghia. All rights reserved.
+//
+
+import Foundation
+import Sapporo
+
+final class DynamicImageSizeCellModel: SACellModel {
+    let title: String
+    
+    init(title: String) {
+        self.title = title
+        //let size = CGSize(width: 320, height: 320)
```

Comment:
Does this needed?

Reply:
I think this class `DynamicImageSizeCellModel`  don't need. Because `DynamicImageSizeCellModel` is never used so I will remove this class